net-mail/poppassd_ceti/poppassd_ceti-1.8.7.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pam
+
+# Long ago it was just "poppassd", but upstream now seems to have
+# settled on "poppassd-ceti" (instead of "poppassd_ceti" or no suffix).
+MY_PN="poppassd-ceti"
+MY_P="${MY_PN}-${PV}"
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Password change daemon with PAM support"
+HOMEPAGE="https://github.com/kravietz/poppassd-ceti"
+SRC_URI="https://github.com/kravietz/${MY_PN}/releases/download/v${PV}/${MY_P}.tar.xz"
+
+LICENSE="GPL-2"
```

Comment:
```
    the Free Software Foundation; either version 2 of the License, or
    (at your option) any later version.```

Reply:
IOW, it's GPL-2+

Reply:
Changed to GPL-2+.

Very minor FYI:
I originally considered changing it on my own, but hadn't
because the following considerations left it slightly less
obvious if I should, yet:
   - The actual 1.8.7 tarball doesn't mention a license at all
     (other than in the auto* compatibility boilerplate stuff; not the
     tool itself).
   - The corresponding git tag DOES include the GPL "LICENSE" file (somehow
     it seemed to be left out of the tarball despite this).
   - The comment at the top of the C file was only modified to mention
     a license (GPL and the "or later version" terminology) after
     the 1.8.7 release.
   - The previous ebuild (1.8.5-r2) just says GPL-2.
   - The previous tarball (1.8.5) actually has a BSDish license file,
     with an extra paragraph saying that you could instead ignore the
     BSD part and use the GPL instead.

Reply:
Oh, I'm sorry. I've looked only at GitHub, and didn't think it would change that recently. Many people are mistakenly using GPL-2 instead of GPL-2+, so I'm a bit sensitive on it.

I think in this case GPL-2+ is a reasonable option but you are free to follow your own judgment.